packages/maskbook/src/social-network/utils/text-payload-ui.ts

```diff
@@ -0,0 +1,21 @@
+import { activatedSocialNetworkUI } from '../ui'
+export function encodePublicKeyUI(x: string): string {
+    const f = activatedSocialNetworkUI.utils.publicKeyEncoding?.encoder
+    if (f) return f(x) || x
```

Comment:
why not is `f(x) ?? x`?

Reply:
If the candidate is `''`, I want to drop that result.